src/main/java/hudson/plugins/claim/ClaimPublisher.java

```diff
@@ -63,11 +74,10 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
             }
         }
 
-        return true;
     }
 
     public BuildStepMonitor getRequiredMonitorService() {
-        return BuildStepMonitor.STEP;
+        return BuildStepMonitor.NONE;
     }
```

Comment:
why are we changing this from STEP to NONE? I could not find any documentation about this Enum in the javadocs